File: cakephp/vristo-cakephp-starter/src/Controller/FormsController.php
<?php
declare(strict_types=1);

namespace App\Controller;

class FormsController extends AppController
{
    public function forms_basic()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/forms/basic');
    }

    public function forms_input_group()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/forms/input-group');
    }

    public function forms_layouts()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/forms/layouts');
    }

    public function forms_validation()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/forms/validation');
    }

    public function forms_input_mask()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/forms/input-mask');
    }

    public function forms_checkbox_radio()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/forms/checkbox-radio');
    }

    public function forms_select2()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/forms/select2');
    }

    public function forms_switches()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/forms/switches');
    }


    public function forms_wizards()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/forms/wizards');
    }

    public function forms_file_upload()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/forms/file-upload');
    }

    public function forms_quill_editor()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/forms/quill-editor');
    }

    public function forms_markdown_editor()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/forms/markdown-editor');
    }

    public function forms_date_picker()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/forms/date-picker');
    }

    public function forms_clipboard()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/forms/clipboard');
    }

    public function forms_touchspin()
    {
        $this->viewBuilder()->setLayout('default');
        return $this->render('/forms/touchspin');
    }
}

File: ruby/vristo-ruby-main/app/controllers/forms_controller.rb
class FormsController < ApplicationController
    def forms_basic
        render layout: "default", :template => 'forms/basic'
    end

    def forms_input_group
        render layout: "default", :template => 'forms/input-group'
    end

    def forms_layouts
        render layout: "default", :template => 'forms/layouts'
    end

    def forms_validation
        render layout: "default", :template => 'forms/validation'
    end

    def forms_input_mask
        render layout: "default", :template => 'forms/input-mask'
    end

    def forms_checkbox_radio
        render layout: "default", :template => 'forms/checkbox-radio'
    end

    def forms_select2
        render layout: "default", :template => 'forms/select2'
    end

    def forms_switches
        render layout: "default", :template => 'forms/switches'
    end

    def forms_wizards
        render layout: "default", :template => 'forms/wizards'
    end

    def forms_file_upload
        render layout: "default", :template => 'forms/file-upload'
    end

    def forms_quill_editor
        render layout: "default", :template => 'forms/quill-editor'
    end

    def forms_markdown_editor
        render layout: "default", :template => 'forms/markdown-editor'
    end

    def forms_date_picker
        render layout: "default", :template => 'forms/date-picker'
    end

    def forms_clipboard
        render layout: "default", :template => 'forms/clipboard'
    end

    def forms_touchspin
        render layout: "default", :template => 'forms/touchspin'
    end
end
